tests/test_experiment_rsmtool_3.py

```diff
@@ -22,6 +27,7 @@
 else:
     from rsmtool.test_utils import rsmtool_test_dir
 
+DIRS_TO_REMOVE = []
```

Comment:
What do you think about using [`tempfile.TemporaryDirectory()`](https://docs.python.org/3.8/library/tempfile.html#tempfile.TemporaryDirectory) that can be explicitly cleaned up after you are done using the `.cleanup()` method? IMO, much better than having global variables and special functions.